core/src/main/java/greencity/exporter/RatingExcelExporter.java

```diff
@@ -0,0 +1,124 @@
+package greencity.exporter;
+
+import greencity.dto.ratingstatistics.RatingStatisticsDto;
+import java.io.IOException;
+import java.util.List;
+import javax.servlet.ServletOutputStream;
+import javax.servlet.http.HttpServletResponse;
+import org.apache.poi.ss.usermodel.Cell;
+import org.apache.poi.ss.usermodel.CellStyle;
+import org.apache.poi.ss.usermodel.Row;
+import org.apache.poi.xssf.usermodel.XSSFFont;
+import org.apache.poi.xssf.usermodel.XSSFSheet;
+import org.apache.poi.xssf.usermodel.XSSFWorkbook;
+
+/**
+ * This class is used for export {@link greencity.entity.RatingStatistics} data to Excel file.
+ *
+ * @author Dovganyuk Taras
+ */
+public class RatingExcelExporter {
+    private XSSFWorkbook workbook;
+    private XSSFSheet sheet;
+    private List<RatingStatisticsDto> ratingStatisticsDtos;
+
+    /**
+     * Constructor.
+     *
+     * @author Dovganyuk Taras
+     */
+    public RatingExcelExporter(List<RatingStatisticsDto> ratingStatisticsDtos) {
+        this.ratingStatisticsDtos = ratingStatisticsDtos;
+        workbook = new XSSFWorkbook();
```

Comment:
This design is not thread save. Imagine the `export()` method is invoked from another thread.